### PR TITLE
fix: remove pro fontawesome, prettify, and fix linting

### PR DESCRIPTION
**Pull request by yousifalraheem**
- rename `seb-` prefix to `sebng-`
- remove pro fontawesome
- prettify
- fix linting

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,7 +1,7 @@
-printWidth: 80
-singleQuote: true
+printWidth: 140
+singleQuote: false
```
Comment (omarVodiaK) [inline .prettierrc]: Please don't :'(

Comment (yousifalraheem) [reply .prettierrc]: What? you wanna keep single quotes? But it doesn't go with any other project we have currently.

Comment (mario-subo) [reply .prettierrc]: YES! Death to single quotes!!! Muahahahaha 

just joking I don't care 👅

File: lib/src/core/selection/selection.ts [hunk 2]
```diff
@@ -6,202 +6,198 @@
...
+    hasValue(): boolean {
+        return !this.isEmpty();
+    }
+
+    /**
+     * Sorts the selected values based on a predicate function.
+     */
+    sort(predicate?: (a: T, b: T) => number): void {
+        if (this._multiple && this.selected) {
+            this._selected && this._selected.sort(predicate);
```
Comment (omarVodiaK) [inline lib/src/core/selection/selection.ts]: i beliece we should start using this syntax
```suggestion
           this._selected?.sort(predicate);
```

Comment (yousifalraheem) [reply lib/src/core/selection/selection.ts]: I know but the Typescript version we have in this project doesn't recognize this syntax. I don't want to go through the process of updating Angular at the moment. At least not in this PR.

File: .prettierrc (hunk 1 above)
Comment (yousifalraheem) [reply .prettierrc]: This is very helpful when you're moving a lot of code around. All our old code is written with double-quotes. It means you're gonna deal with a lot of unnecessary linting issues.

File: projects/ng-components-docs/src/app/components/header/header.component.html [hunk 3]
```diff
@@ -1,40 +1,43 @@
...
-        <div class="col-auto px-0">
-          <app-logo></app-logo>
+    <div class="row justify-content-between justify-content-lg-end h-100">
+        <div class="col col-md-auto d-none d-md-flex brand">
+            <div class="row mobile-header">
+                <div class="col-auto px-0">
+                    <app-logo></app-logo>
+                </div>
+                <a class="col-auto align-self-center text-center px-3 text-light" href="#" (click)="toggleMenu($event)">
+                    Menu
```
Comment (omarVodiaK) [inline projects/ng-components-docs/src/app/components/header/header.component.html]: What will happen to the icon ?

File: lib/src/core/selection/selection.ts (hunk 2 above)
Comment (omarVodiaK) [reply lib/src/core/selection/selection.ts]: I've seen this syntax in some other file lib/src/modal/modal.ts

**Pull request merged**